### ISSUE TITLE
No module found for 'ApplicationServices.Swift' with Python3 on macOS 12 Monterey

### ISSUE THREAD

**Issue by yuleyl** [ronaldoussoren/pyobjc#378]

**Comment by ronaldoussoren** [comment]
Could you provide more information?  There is no ApplicationServices.Swift module.  Do you get an error about this when importing a PyObjC module/package?

Also:
- Which Python version?
- Which PyObjC version?

**Comment by ronaldoussoren** [comment]
There are no references to ApplicationServices.Swift in the PyObjC sources.

I will close this issue unless you provide more information that I can use to reproduce.

**Comment by ronaldoussoren** [comment]
I can reproduce the issue with top of the tree:

```
>>> from ApplicationServices import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'ApplicationServices.Swift'
```

**Comment by ronaldoussoren** [comment]
Thanks for your report. The issue will be fixed in the 8.0 release.

In general I advise to avoid star imports (e.g. ``from Framework import *`` because framework bindings expose a lot of names and that causes those imports to be close. Just importing what you need tends to be a lot faster, and is also easier
when using code quality tools like flake8.
